src/server_manager/electron_app/digitalocean_oauth.ts

```diff
@@ -209,6 +208,9 @@ export function runOauth(): OauthSession {
         }
         reject(error);
       });
+
+    // Automatically close the server after 30 seconds.
+    setTimeout(server.close, 30000);
```

Comment:
Will this send an error/do we want to send a 408? Also, mind moving the 30000 to a const at the head of the file or something, I'm iffy on magic nos

Reply:
This is inside the POST handler for the oauth response, so the error response would go to whoever sent the improper request (or we would return 200 success for the request + correct token from digital ocean).

But that brings up a good point, I should actually move this timeout to right after the server creation, instead of in the POST handler.  I'll also add a const definition for the timeout.